src/main/java/graphql/language/DirectivesContainer.java

```diff
@@ -34,20 +34,16 @@ public interface DirectivesContainer<T extends DirectivesContainer> extends Node
      *
```

Comment:
We no longer have default methods - but rather a memory efficient DirectivesHolder helper